src/index.html

```diff
@@ -1,13 +1,460 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Miami</title>
+    <title>MyBiKE</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
     <script src="scripts/main.js" defer></script>
+    <link rel="icon" href="./images/favicon.svg">
   </head>
-  <body>
-    <h1>Miami</h1>
+  <body class="page__body">
+    <header class="page__header header" >
+      <div class="container">
+        <div class="header__content">
+          <div class="header__top">
+            <a
+              href="#"
+              class="logo">
+              <img
+                class="logo__image"
+                src="./images/logo.svg"
+                alt="MyBiKE logo"
+              >
+            </a>
+
+            <div class="header__actions">
+              <a
+                href="tel:+1 234 5555-55-55"
+                class="
+                  header__icon
+                  icon
+                  icon--phone"
+              >
+                <div class="icon__tooltip">
+                  +1 234 5555-55-55
+                </div>
+              </a>
+
+              <a
+                href="#menu"
+                class="
+                  header__icon
+                  icon
+                  icon--menu"
+                ></a>
+            </div>
+          </div>
+
+          <h1 class="header__title">Take the Streets</h1>
+        </div>
+      </div>
+    </header>
+
+    <nav class="page__menu menu" id="menu">
+      <div class="container">
+        <div class="menu__content">
+          <div class="menu__top">
+            <a href="#" class="logo">
+              <img
+                class="logo__image"
+                src="./images/logo_dark.svg"
+                alt="MyBiKE logo">
```

Comment:
put close tag to next row
```suggestion
                alt="MyBiKE logo"
               >
```

Reply:
+